src/Scripts.purs

```diff
@@ -86,17 +86,17 @@ validatorHashAddress networkId =
   baseAddressToAddress <<< validatorHashBaseAddress networkId
 
 -- | Converts a Plutus-style `StakeValidator` to an `Address`
-stakeValidatorHash :: StakeValidator -> Maybe StakeValidatorHash
+stakeValidatorHash :: StakeValidator -> QueryM (Maybe StakeValidatorHash)
 stakeValidatorHash = plutusScriptHash
 
 plutusScriptHash
   :: forall (m :: Type) (n :: Type)
    . Newtype m PlutusScript
   => Newtype n ScriptHash
   => m
-  -> Maybe n
-plutusScriptHash = map wrap <<< scriptHash <<< unwrap
+  -> QueryM (Maybe n)
+plutusScriptHash = map (map wrap) <<< scriptHash <<< unwrap
 
 -- | Converts a `PlutusScript` to a `ScriptHash`.
-scriptHash :: PlutusScript -> Maybe ScriptHash
-scriptHash = scriptHashFromBytes <<< unwrap
+scriptHash :: PlutusScript -> QueryM (Maybe ScriptHash)
+scriptHash = undefined
```

Comment:
Intended?

Reply:
Yes, I'm going to fix this in a separate PR